contributors/design-proposals/vertical-pod-autoscaler.md

```diff
@@ -0,0 +1,486 @@
+Vertical Pod Autoscaler
+=======================
+Vertical Pod Autoscaler (VPA, aka. "rightsizing" or "autopilot") is an
```

Comment:
The main issue for this is https://github.com/kubernetes/kubernetes/issues/10782